cfme/tests/automate/custom_button/test_generic_objects.py

```diff
@@ -315,6 +315,7 @@ def test_custom_button_expression_evm_obj(appliance, request, setup_obj, button_
 
 
 @pytest.mark.manual
+@pytest.mark.ignore_stream("5.10")
 def test_custom_button_open_url_evm_obj(setup_obj, button_group):
```

Comment:
@digitronik looks like this test and `test_custom_button_open_url_service_obj` are new features, can you also add `coverage` to them? I think it could be helpful.

Reply:
@lina-nikiforova it was not come under scope of PR; but yup, it good to update this :+1:

Reply:
@digitronik agreed it's out of scope :) thanks :)